pallets/delegation/src/mock.rs

```diff
@@ -27,6 +27,7 @@ use frame_support::{
 };
 use frame_system::EnsureSigned;
 use kilt_primitives::constants::delegation::DELEGATION_DEPOSIT;
+use kilt_support::deposit::Deposit;
```

Comment:
revert this

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/267/commits/1f16749100032c3759b6c948ee6d96c7ad95a476